BE/PoroDev.Runtime/Services/DockerImageService.cs

```diff
@@ -247,7 +247,7 @@ public async Task<DockerRuntimeException> GetProcessedOutput(Guid userId)
 
                 byte[] fileBytes = File.ReadAllBytes(Path.Combine(Directory.GetParent(Environment.CurrentDirectory).FullName, "imageEdited.jpg"));
 
-                await _uploadClient.GetResponse<CommunicationModel<FileUploadResponse>>(new { FileName = $"Runtime Output {DateTime.UtcNow}",
+                await _uploadClient.GetResponse<CommunicationModel<FileUploadResponse>>(new { FileName = $"Runtime Output {DateTime.UtcNow}.jpg",
                                                                                               File = fileBytes, 
                                                                                               ContentType = "image/jpeg", 
```

Comment:
Please change contentType here to jpg.
Also, could we have ".jpg" extension (and others: png, jpeg,png...) be a constant here?
Do we have a way of knowing what extensions type should we return (similarly, can we dynamically set extensions here)?